tests/sentry/quotas/test_redis.py

```diff
@@ -148,6 +149,14 @@ def test_abuse_quotas(self):
         assert quotas[4].window == 10
         assert quotas[4].reason_code == "org_abuse_limit"
 
+        assert quotas[5].id == "gam"
+        assert quotas[5].scope == QuotaScope.GLOBAL
+        assert quotas[5].scope_id is None
+        assert quotas[5].categories == {DataCategory.METRIC_BUCKET}
+        assert quotas[5].limit == 6040
```

Comment:
Above here I set the limit to 604.

`self.organization.update_option("global-abuse-quota.metric-bucket-limit", 604)`

since the abuse_window is 10 seconds its multiplied by 10